Sources/TuistCloud/Services/CloudURLService.swift

```diff
@@ -29,7 +29,7 @@ public final class CloudURLService: CloudURLServicing {
     public init() {}
 
     public func url(serverURL: String?) throws -> URL {
-        let cloudURL = serverURL ?? ProcessInfo.processInfo.environment["TUIST_CLOUD_URL"] ?? Constants.tuistCloudURL
+        let cloudURL = ProcessInfo.processInfo.environment["TUIST_CLOUD_URL"] ?? serverURL ?? Constants.tuistCloudURL
```

Comment:
`TUIST_CLOUD_URL` is defined as a constant, so if we keep it, let's use `Constants.cloudURL`